script/github-actions/sasslint-annotation-format.js

```diff
@@ -0,0 +1,26 @@
+// imports dynamically created file
+const sasslintData = require('../../sasslint-report.json'); // eslint-disable-line import/no-unresolved
+
+for (let i = 0; i < sasslintData.length; i++) {
+  if (sasslintData[i].warningCount > 0 || sasslintData[i].errorCount > 0) {
+    for (let j = 0; j < sasslintData[i].messages.length; j++) {
+      let output = '';
+
+      if (sasslintData[i].messages[j].severity === 0) {
+        output += '::error ';
+      } else {
+        output += '::warning ';
+      }
+
+      output += `file=${sasslintData[i].filePath},line=${
+        sasslintData[i].messages[j].line
+      },col=${sasslintData[i].messages[j].column}::${
+        sasslintData[i].filePath
+      }:${sasslintData[i].messages[j].line}:${
+        sasslintData[i].messages[j].column
+      }:${sasslintData[i].messages[j].message}`;
```

Comment:
Could this be made more readable with `forEach` and destructuring? I'm thinking something like:

```
sassLintData.forEach(fileMessages => {
  if (!fileMessages.warningCount && !fileMessages.errorCount) return;

  fileMessages.forEach(fileMessage => {
    const {
      severity,
      filePath,
      column,
      line,
      message,
    } = fileMessage;

    const severityTag = severity === 0 ? '::error' : '::warning';
    const output = `${severityTag} file=${filePath},line=${line},col=${column}::${filePath}:${line}:${column}:${message}`;
    console.log(output);
  });
});
```